Dockerfile.template

```diff
@@ -0,0 +1,30 @@
+# install a Debian image with Python 3 installed
+FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3-stretch-run
+
+RUN install_packages apt-utils
+RUN install_packages i2c-tools
+RUN install_packages python3-smbus
+
+# use apt-get if you need to install dependencies
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+	python3-envirophat \
+	libraspberrypi-bin
```

Comment:
similarly would add these into the above ^^

Reply:
@shaunmulligan aren't these already one line?